Assignment4/src/controllers/UserControoler.js

```diff
@@ -0,0 +1,13 @@
+const { secretKey, mockData } = require("../utils/helperFunction");
```

Comment:
secret should always come from env file via config.

Reply:
Updated